docs/content/docs/data-structures/overview.md

```diff
@@ -0,0 +1,105 @@
+
+* [SharedMap]({{< relref "/docs/apis/map/sharedmap.md" >}}) - a basic key-value data structure
+* [SharedDirectory]({{< relref "/docs/apis/map/shareddirectory.md" >}}) – a SharedMap with hierarchical paths instead of
+simple keys
+* [SharedCell]({{< relref "/docs/apis/cell/sharedcell.md" >}}) – a “single-object SharedMap”; useful for wrapping objects.
+
+### Key Value Scenarios
+
+Key-value data structures are the most common choice for many scenarios.
+
+* user preference data
+* current state of a survey
+* the configuration of a view
+
+### Common Issues
+
+* Storing a counter in a map will have unexpected behavior. Use the SharedCounter instead.
+* Storing a lot of data in one key-value entry may cause performance issues. Each update will update the entire value.
+Try splitting the data across multiple keys
+* Storing arrays, lists, or logs in a key-value entry may lead to unexpected behavior because users can't collaboratively
+modify parts of one entry. Try storing the array or list data in a SharedSequence or SharedInk
+
+## Sequences
+
+These DDSes are used for storing sequential data. They are optimistic.
+
+* [SharedNumberSequence]({{< relref "SharedNumberSequence" >}}) – a sequence of numbers.
+* [SharedObjectSequence]({{< relref "/docs/apis/sequence/sharedobjectsequence.md" >}}) – a sequence of objects.
+* [SharedMatrix]({{< relref "SharedMatrix" >}}) – a data structure to efficiently use two-dimensional tabular data.
+* [SharedString]({{< relref "SharedString" >}}) – a specialized data structure for handling collaborative text.
+
+Sequence Data Structures are useful when you'll need to insert data in the middle of a list or array. Unlike the
+key-value data structures, sequences have a sequential order and can handle simultaneous inserts from multiple users.
+
+### Sequence Scenarios
+
+* Text editors
+* Tabular data
+* Timelines
+* Lists
+
+## Specialized data structures
+
+* [SharedCounter]({{< relref "SharedCounter" >}}) – a counter.
+
+    The SharedCounter is useful to keep track of increments. While a key-value data structure appears like a good fit,
+    two users simultaneously setting the same key can cause issues.
+
+* [Ink]({{< relref "/docs/apis/ink/ink.md" >}}) – a specialized data structure for ink data.
+
+    Ink is a specific form of an append only list. It's great for capturing ink strokes.
+
+
+## Consensus Data Structures
+
+These DDSes are **not** optimistic. Before a change to a consensus data structure is confirmed, the connected clients
+must acknowledge the change.
+
+* [OrderedCollection]({{< relref "/docs/apis/ordered-collection" >}}) - an ordered queue of items, but each item is pulled
+off the queue by only one client
+* [RegisterCollection]({{< relref "/docs/apis/register-collection" >}}) - Stores values, but keeps a record of all changes
+* Quorum - Allows clients to agree on a proposal. Quorum also contains client information
```

Comment:
```suggestion
* [OrderedCollection]({{< relref "/docs/apis/ordered-collection" >}}) -- an ordered queue of items, but each item is pulled
off the queue by only one client.
* [RegisterCollection]({{< relref "/docs/apis/register-collection" >}}) -- Stores values, but keeps a record of all changes.
* Quorum -- Allows clients to agree on a proposal. Quorum also contains client information.
```

Periods at the end of list items and `--` for an en-dash instead of hyphen.

Reply:
here and elsewhere